tests/test_default_project.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict, Text
+
+from warnings import WarningMessage
+from _pytest.pytester import Testdir
+import pytest
+
+from rasa.__main__ import create_argument_parser
+import rasa.cli.data
+import rasa.cli.scaffold
+import rasa.cli.train
+import rasa.shared.utils.io
+
+
+def _warning_should_be_filtered_out(warning: WarningMessage) -> bool:
+    # we filter out `gelu` warnings because of this issue:
+    # https://github.com/RasaHQ/rasa/issues/9129
+    # this function can be removed once we migrate to TensorFlow 2.6
+    return type(warning.message) == DeprecationWarning and str(
+        warning.message
+    ).startswith("gelu activation has been migrated to core TensorFlow")
+
+
+@pytest.mark.timeout(300, func_only=True)
+def test_default_project_has_no_warnings(
+    testdir: Testdir, default_config: Dict[Text, Any]
+):
+    parser = create_argument_parser()
+    rasa.cli.scaffold.create_initial_project(".")
```

Comment:
Can we lower the amount of epochs to speed up the test? That would be a compromise between speed and maintaining the default project as it is